tests/foreman/api/test_errata.py

```diff
@@ -199,17 +199,21 @@ def test_positive_install_multiple_in_host(
             'activationkey-id': ak.id,
         }
     )
-    # Install katello-ca, register content-host, enable all repos:
-    rhel_contenthost.install_katello_ca(target_sat)
-    rhel_contenthost.register_contenthost(function_org.name, ak.name, function_lce.name)
+    rhel_contenthost.register(
+        org=function_org,
+        activation_keys=ak.name,
+        lifecycle_environment=function_lce,
+        target=target_sat,
+        force=True,
+        loc=None,
```

Comment:
just a nitpick, can you keep both org and loc args together

Reply:
syntax issue, arguments set to None have to be after any explicit arguments.

Reply:
yeah, so we argument for org can be kept at last